CMakeLists.txt

```diff
@@ -48,12 +48,12 @@ if(Boost_FOUND)
 endif()
 add_subdirectory(doc)
 
+set(CPACK_PACKAGE_VENDOR "bluebrain.epfl.ch")
+set(CPACK_DEBIAN_PACKAGE_DEPENDS "qtbase5-dev, libturbojpeg")
```

Comment:
It is, tested on a fresh Ubuntu 16.04 VM :-)
Only simplestreamer uses glut, but since it is just a dummy test application I don't think it's worth including.